### ISSUE TITLE
Since we are out of travis builds, we should probably move to GitHub actions

### ISSUE THREAD

**Issue by SylvainCorlay** [jupyter/nbconvert#1615]
Currently, travis CI does not run.

**Comment by mgeier** [comment]
The Travis-CI people have a strange policy regarding offering free services for OSS projects: each project has to individually ask them for free minutes, see also https://travis-ci.community/t/builds-fully-hanging/10613/4.

I've contacted their support once and they have given me some random amount of free minutes. I'll have to ask again once they are all consumed.

This is an excerpt from one of their e-mails:

> We offer an Open Source Subscription for free to non-commercial open-source projects. To qualify for an Open Source subscription, the project must meet the following requirements:
>
>  *  You are a project lead or regular committer (latest commit in the last month)
>  *  Project must be at least 3 months old and is in active development (with regular commits and activity)
>  *  Project meets the OSD specification
>  *  Project must not be an expressly commercial project
>  *  Project can not provide commercial services or distribute paid versions of the software
>
> Does this sound like you and your project? We'd be very happy to support you!

Another excerpt from another e-mail:

> Open source has always been and always will be at the core of what Travis CI stands for.

OSS is at their core, and they are happy to support us, but they still make us beg for free credits.

Long story short: you'll get free minutes if you ask for it, but switching to Github Actions is probably less cumbersome in the long run.

**Comment by jasongrout** [comment]
I also took a crack at this over in #1617.
